controllers/rhmi/bootstrapReconciler.go

```diff
@@ -296,8 +296,22 @@ func (r *Reconciler) deleteObsoleteService(ctx context.Context, serverClient k8s
 			}
 		}
 	}
+	
+	if r.installation.Spec.Type == string(integreatlyv1alpha1.InstallationTypeManagedApi) {
```

Comment:
looks good but I would refactor the above "if" instead as it's no longer required.

Reply:
We can also remove the comment on 285 and get a follow up jira to remove this once all clusters have been bumped to 1.27

Reply:
The above 'if' is due to be removed by this PR https://github.com/integr8ly/integreatly-operator/pull/2714/files#diff-c65c68fc76da4477e6673d7e7dd25a1d0d971a7be076457cd8f445b3853e96b7L285
I can add a comment to this new if, and create the jira ticket. Thank you

Reply:
Existing IF refactored. Follow up Jira created https://issues.redhat.com/browse/MGDAPI-4582.